plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -22,13 +22,17 @@ deploy.preferences.save.error.title=Could not save preferences
 deploy.preferences.save.error.message=Error while saving the preferences: {0}
 error.account.missing.signedin=Select an account.
 error.account.missing.signedout=Sign in to Google.
+error.not.a.file=Not a file: {0}
```

Comment:
"Is a directory, not a file" (assuming that's accurate here)

Reply:
It's not so clear if it has to be a directory, according to the Javadoc of `File.isFile()`. I will update it to "Path is a directory or not a normal file: {0}".
```
     * Tests whether the file denoted by this abstract pathname is a normal
     * file.  A file is <em>normal</em> if it is not a directory and, in
     * addition, satisfies other system-dependent criteria.  Any non-directory
     * file created by a Java application is guaranteed to be a normal file.
```